src/main/java/seedu/address/logic/commands/UpdateCommand.java

```diff
@@ -53,6 +54,8 @@ public class UpdateCommand extends Command {
 
     public static final String MESSAGE_INDEX_OUT_OF_BOUNDS = "There is no event at this index";
 
+    public static final String ERR_END_DATE_BEFORE_START_DATE = "The start date must come before the end date";
```

Comment:
Perhaps we should rename it `MESSAGE_ERR_END_DATE_BEFORE_START_DATE` or something along those lines, to be consistent with `MESSAGE_INDEX_OUT_OF_BOUNDS` ?